src/connectedvmware/azext_connectedvmware/_params.py

```diff
@@ -346,12 +352,77 @@ def load_arguments(self, _):
             help="Username to use for connecting to the vm.",
         )
         c.argument(
-            'password',
-            options_list=['--password'],
+            'password', options_list=['--password'],
+            help="Username password credentials to use for connecting to the VM.",
+        )
+
+    with self.argument_context('connectedvmware vm guest-agent show') as c:
+        c.argument(
+            'password', options_list=['--password'],
             help="Username password credentials to use for connecting to the VM.",
         )
 
     with self.argument_context('connectedvmware vm guest-agent show') as c:
         c.argument(
             'vm_name', options_list=['--vm-name'], help="Name of the VM.",
         )
+
+    with self.argument_context('connectedvmware vm extension list') as c:
+        c.argument('vm_name', options_list=['--vm-name'], type=str, help='The name of the vm containing the extension.')
+        c.argument('expand', type=str, help='The expand expression to apply on the operation.')
+
+    with self.argument_context('connectedvmware vm extension show') as c:
+        c.argument(
+            'vm_name', options_list=['--vm-name'], type=str, help='The name of the vm containing the extension.',
+            id_part='name')
+        c.argument('name', type=str, help='The name of the vm extension.', id_part='child_name_1')
+
+    with self.argument_context('connectedvmware vm extension create') as c:
+        c.argument(
+            'vm_name', options_list=['--vm-name'], type=str, help='The name of the vm where the extension '
+            'should be created or updated.')
+        c.argument('name', type=str, help='The name of the vm extension.')
+        c.argument('tags', tags_type)
+        c.argument('force_update_tag', type=str, help='How the extension handler should be forced to update even if '
+                   'the extension configuration has not changed.')
+        c.argument('publisher', type=str, help='The name of the extension handler publisher.')
+        c.argument('type_', options_list=['--type'], type=str, help='Specifies the type of the extension; an example '
+                   'is "CustomScriptExtension".')
+        c.argument('type_handler_version', type=str, help='Specifies the version of the script handler.')
```

Comment:
Please use the first person voice
```suggestion
        c.argument('type_', options_list=['--type'], type=str, help='Specify the type of the extension; an example '
                   'is "CustomScriptExtension".')
        c.argument('type_handler_version', type=str, help='Specify the version of the script handler.')
```